starters/vue3-xstate-css/package.json

```diff
@@ -1,15 +1,19 @@
 {
 	"name": "vue3-xstate-css",
 	"version": "0.0.0",
-	"description": "Vue 3, XState, and CSS",
+	"description": "Vue3, XState, and CSS",
 	"keywords": [
 		"vue",
 		"xstate",
 		"css"
 	],
 	"hasShowcase": false,
+	"engines": {
+		"node": "16.18"
```

Comment:
how come this has to be added?
Isn't this covered in the nvmrc file

Reply:
I added this as an extra precaution, so it helps make sure we use the right node version. It works alongside the .npmrc file to give an error if you're on the wrong node version.